site/content/docs/0.0/getting-started/javascript.md

```diff
@@ -8,4 +8,312 @@ aliases:
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Individual or compiled
+
+Plugins can be included individually (using OUDS Web's individual `js/dist/*.js`), or all at once using `ouds-web.js` or the minified `ouds-web.min.js` (don't include both).
+
+If you use a bundler (Webpack, Parcel, Vite...), you can use `/js/dist/*.js` files which are UMD ready.
+
+## Usage with JavaScript frameworks
+
+While the OUDS Web CSS can be used with any framework, **the OUDS Web JavaScript is not fully compatible with JavaScript frameworks like React, Vue, and Angular** which assume full knowledge of the DOM. Both OUDS Web and the framework may attempt to mutate the same DOM element, resulting in bugs like dropdowns that are stuck in the "open" position.
+
+A better alternative for those using this type of frameworks is to use a framework-specific package **instead of** the OUDS Web JavaScript. Here are some of the most popular options:
+
+- React: [React Bootstrap](https://react-bootstrap.github.io/)
+  {{< callout >}}
+  **Try it yourself!** Download the source code and working demo for using Bootstrap with React, Next.js, and React Bootstrap from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/react-nextjs). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/react-nextjs?file=src%2Fpages%2Findex.tsx).
+  {{< /callout >}}
+<!--- Vue: [BootstrapVue](https://bootstrap-vue.org/) (Bootstrap 4)-->
+- Vue 3: [BootstrapVueNext](https://bootstrap-vue-next.github.io/bootstrap-vue-next/) (Bootstrap 5, currently in alpha)
+- Angular: [ng-bootstrap](https://ng-bootstrap.github.io/)
+
+## Using OUDS Web as a module
+
+{{< callout >}}
+**Try it yourself!** Download the source code and working demo for using Bootstrap as an ES module from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/sass-js-esm). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/sass-js-esm?file=index.html).
+{{< /callout >}}
+
+{{< callout warning >}}
+**To get a working OUDS Web demo based on the Bootstrap ones**, you need to replace all `bootstrap` occurrences with `ouds-web` in `scss/style.scss`, within `<script>` tags in `index.html` and in `package.json`.
```

Comment:
Yeah, you're right, the paragraph should be improved to mention at least that:
- In JavaScript, it'll be "oudsWeb" rather than "ouds-web"
- Mention that for a specific Bootstrap version, one should load the "ouds-web-with-bootstrap" files

Reply:
Tried something:

```
{{< callout warning >}}
**To set up a functional OUDS Web demo based on the Bootstrap templates**, follow these steps to replace all `bootstrap` references:
* Change `bootstrap` to `ouds-web` in `scss/style.scss` and within `import` statements in `index.html`.
* Replace `bootstrap` with `@ouds/web` in `package.json`, and within `import` statements in `index.html` for the CDN URL.
* Update JavaScript objects from `bootstrap` to `oudsWeb`.
{{< /callout >}}
```

What do you think?

Reply:
TBH, I really like it, let's go with it, thanks for the wording work!